ios/tools/prepRelease.sh

```diff
@@ -71,13 +73,13 @@ echo "engine dest: $KMEI_DST"
 
 echo "Zipping ${FRAMEWORK} => ${UPLOAD_DIR}/${KMEI_DST}..."
 cd "${KMEI_FRAMEWORK_BASE}"
-zip -qrX "${KMEI_DST}" ${FRAMEWORK}
+zip_files "${KMEI_DST}" "-qrX" "${FRAMEWORK}"
```

Comment:
Would we ever have 7z do this?
It gives an error 

```
./prepRelease.sh
Writing changelog to upload/19.0.40/changelog-19.0.40-alpha.txt
engine dest: /c/src/keymanapp/ios/upload/19.0.40/keyman-engine-ios-19.0.40.zip
Zipping KeymanEngine.xcframework => upload/19.0.40//c/src/keymanapp/ios/upload/19.0.40/keyman-engine-ios-19.0.40.zip...
[ios/tools] [DEBUG] C:\Program Files\7-Zip/7z.exe a -qrX /c/src/keymanapp/ios/upload/19.0.40/keyman-engine-ios-19.0.40.zip KeymanEngine.xcframework


Command Line Error:
Unknown switch:
-qrX
```

Maybe we split the flags `-q`, `-r`, `-X` and translate the flags that apply to 7z?

Reply:
Yes, sounds good to me. Not sure what the flags are for but you can figure that out I am sure :grin: